test_conformance/CMakeLists.txt

```diff
@@ -89,3 +89,13 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+# Copy all csv files
+FILE(GLOB CSV_FILES
+ 	 "${CMAKE_CURRENT_SOURCE_DIR}/*.csv"
+)
+FILE(COPY ${CSV_FILES} DESTINATION ".")
```

Comment:
@kpet  For the current working directory I couldn't any variables. I mean to specify the destination directory.

Reply:
> @kpet For the current working directory I couldn't any variables. I mean to specify the destination directory.

fixed it by using "CMAKE_BINARY_DIR" variable